src/repositories/blockAddressPointOfLp.repository.ts

```diff
@@ -22,6 +23,15 @@ export class BlockAddressPointOfLpRepository extends BaseRepository<BlockAddress
     return result;
   }
 
+  public async getUniquePointKeyByBlockRange(startBlock: number, endBlock: number, type: string) {
+    const transactionManager = this.unitOfWork.getTransactionManager();
+    const blockAddressPoints = await transactionManager.getRepository(BlockAddressPointOfLp).find({
+      where: { blockNumber: Between(startBlock, endBlock), type: type },
+      select: ["address", "pairAddress"],
+    });
+    return new Set(blockAddressPoints.map((point) => `${point.address}-${point.pairAddress}`));
```

Comment:
Why isn't the key here using blockNumber?

Reply:
good point. Not adding blockNumber here shouldn't cause any issues because the data range is already specified in the query conditions. Of course, adding blockNumber here is also fine and would make the code clearer to understand.